frontend_lib/test/component/ProgressBar.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import ProgressBar from '../../src/component/ProgressBar/ProgressBar.jsx'
+
+describe('<ProgressBar />', () => {
+  const props = {
+    percent: 50,
+    backgroundColor: 'yellow',
+    color: 'red'
+  }
+
+  const wrapper = shallow(
+    <ProgressBar
+      {...props}
+    />
+  )
+
+  describe('Static design', () => {
+    it(`should display its background in color ${props.backgroundColor}`, () =>
+      expect(wrapper.find(`.progress-value`).prop('style').backgroundColor).to.equal(props.backgroundColor)
+    )
+
+    it(`should display the text ${props.percent}`, () =>
+      expect(wrapper.find(`.progress-value`)).to.have.lengthOf(1)
```

Comment:
here you are only testing that the class exists in the DOM, not that the text is the right percent